scripts/install

```diff
@@ -174,15 +176,16 @@ FTL_temp_path=$(mktemp -d)
 ynh_setup_source "$FTL_temp_path" FTL
 
 # Plutôt que télécharger le binaire C, on le compile nous-même.
-( cd "$FTL_temp_path"
-ynh_exec_warn_less make
-ynh_exec_warn_less make install )
-ynh_secure_remove "$FTL_temp_path"
+#( cd "$FTL_temp_path"
+#ynh_exec_warn_less make
+#ynh_exec_warn_less make install )
+#ynh_secure_remove "$FTL_temp_path"
+apt install -y dnsmasq
```

Comment:
Without make, FTL will not be installed.
dnsmasq is already installed, as it's the DNS resolver of YunoHost.

Reply:
ok have error on make have # make for past error

Reply:
Sure, commenting a line that produce an error is a good way to hide an error. But that's not going to fix anything, just hiding the issue ;)

Reply:
We're going to have a problem with that anyway...
https://github.com/pi-hole/FTL#installation

    FTLDNS will disable any existing installations of dnsmasq. This is because FTLDNS is dnsmasq + Pi-hole's code, so both cannot run simultaneously.

Which means we can't use the last versions that use that version of FTL. Because we can't disable our current version of dnsmasq, which is used by YunoHost.
So, we have to find the last working version of Pi-Hole for us, and stay stick to it...

Reply:
- I just checked, FTL 2.13.2 is the last available version we can use with jessie. The version 3.0 isn't compatible with the dnsmasq version available on jessie.
And the 4.0 version isn't working, as explained just before...
- AdminLTE 3.3 is the version that match Pi-Hole 3.3.1
- And Pi-Hole 3.3.1 is probably the last version that works without FTL 4.0. But I'm going to try that right now.

Reply:
im use strech

Reply:
You maybe, but not all our users. So we (at least I) want to keep a back compatibility to Jessie.

Reply:
Can you speak in french ?

Reply:
As expected, Pi-Hole 4.0 doesn't work with FTL 2.13.2... Which means to upgrade to Pi-Hole 4.0 we should use FTL 4.0 with it, which we can't do.
So I guess that means we're stuck with this current version of Pi-Hole.

Reply:
> Can you speak in french ?

Unfortunately not...
But, is there anything you don't understand, I can try to explain it a different way.

Reply:
no is ok for english me but need FTL 4.0 you have possibility to help me for fix that on my fork ?
For me is not a problem no use dnsmasq just need login page and FTL 4.0 / Pi-hole 4.0

Reply:
dnsmasq is used by YunoHost, you can't disable it and replace it by another version without be really sure it's not going to have any effects on YunoHost.

Why do you need the version 4.0 ?

Reply:
In my enterprise im using fonctionallity on 4.0 not on 3.0 and have other problem im waiting 4.1 version this version add better support for my work and fix important bugs for me !
I need this version 4.0 minimal no less :( but i love yunohost and for my is important to add this version on yunohost for yunohost more in my enterprise

Reply:
And did you try to install Pi-Hole directly ? Without the app ?

Reply:
Looks like the make method doesn't work anymore... And the install script uses directly the binary file
https://github.com/pi-hole/pi-hole/blob/master/automated%20install/basic-install.sh#L2070-L2075

You can try that way if you prefer.

Reply:
https://paste.yunohost.org/epifayucoc.sql ok have try but need more information for fix that !

Reply:
Yes but just need to disable dnsmasq for run FTL 4.0

Reply:
You can help for fix that on this script install and have yunohost dev server for test after fix that !
Please is important for me to try that

Reply:
The script is supposed to do it by itself.
Is there any verbose mode for this script ?

Reply:
im try in verbose mode

Reply:
no have found verbose mode :(

Reply:
have read  /etc/pihole/install.log is same message

Reply:
have fix 2 errors :D